src/main.js

```diff
@@ -66,14 +66,15 @@ export default class Auth {
 		// we need to check if this environment supports `addEventListener` on the window.
 		'addEventListener' in window &&
 			window.addEventListener('storage', e => {
-				// This code will run if localStorage for this user
-				// data was updated from a different browser window.
+				// This code will run if the local storage for this user
```

Comment:
The meaning was literally “localStorage” as in the API, so this one needs to remain as it was.